script.js

```diff
@@ -0,0 +1,22 @@
+
+fetch("products.json")
+.then(function(response){
```

Comment:
why are you fetching the data from JSON file , you should be calling the real API , please change the implementation